src/js/view/view.js

```diff
@@ -855,7 +863,7 @@ function View(_api, _model) {
         if (floatingPlayer === _playerElement) {
             floatingPlayer = null;
 
-            removeClass(_playerElement, 'jw-flag-floating');
+            removeClass(_playerElement, 'jw-flag-floating jw-flag-small-player');
```

Comment:
Do we want to do this? Or do we want to do
`removeClass(_wrapperElement, 'jw-flag-small-player')`?